src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -45,14 +45,24 @@ private static X509Certificate2Collection GetCertificateCollection(string certif
             {
                 // On Windows, .Net Core applications should not import PFX files in parallel to avoid a known system-level race condition.
                 // This bug results in corrupting the X509Certificate2 certificate state.
-                Assert.True(m.WaitOne(MutexTimeout), "Cannot acquire the global certificate mutex.");
-
-                var certCollection = new X509Certificate2Collection();
-                certCollection.Import(Path.Combine(TestDataFolder, certificateFileName), CertificatePassword, X509KeyStorageFlags.DefaultKeySet);
+                try
+                {
+                    Assert.True(m.WaitOne(MutexTimeout), "Cannot acquire the global certificate mutex.");
```

Comment:
I think it would be good to change, as hiding assertion failure exceptions makes the test more difficult to diagnose, though not by much in this case. Probably not critical but maybe as cleanup.